scripts/inner/kubestellar-kube-bind

```diff
@@ -85,17 +86,22 @@ if kubectl get crd $resource_to_bind.edge.kubestellar.io &> /dev/null; then
     exit 0
 fi
 
+if [[ ! -d $log_dir ]]; then
```

Comment:
Isn't this what the `-p` flag on `mkdir` means?

Reply:
Removed this in #1351.